CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Added
 
+- Remove libxdg-basedir dependency (GLib's function is used instead)
```

Comment:
> GLib's g_get_user_config_dir function does exactly the same thing

Unfortunately for us, it doesn't. Debian places the default config in /etc/xdg/dunst/dunstrc which `libxdg-basedir` loads properly, but `g_get_user_config_dir` does not.

Reply:
Good point. I've rewritten it and it'll respect now the system dirs, too!